Fundamentals/Code/Source-Control/index-v9.md

```diff
@@ -0,0 +1,174 @@
+---
+versionFrom: 9.0.0
+meta.Title: "Source Control with Umbraco"
+meta.Description: "In this article you can learn more about how to effectively source control your Umbraco site."
+state: complete
+verified-against: rc-003
+update-links: false
+---
+
+# Source Control
+
+## Umbraco Cloud
+
+If you are running your site on Umbraco Cloud - and why wouldn't you be? - then source control is very much part of the experience, have a look at the ['Technical overview of an Umbraco Cloud Environment'](../../../Umbraco-Cloud/Getting-Started/Environments/) and the information on ['Working with your Umbraco Cloud project'](../../../Umbraco-Cloud/Set-Up/#working-with-your-umbraco-cloud-project) for a steer on Source/Version Control good practices.
+
+## Outside of Umbraco Cloud
+
+If you are hosting your Umbraco implementation outside of Umbraco Cloud, it's generally considered good practice to setup source/version control for your site implementation files. This is especially a good idea when you are working with a team as it can help you track changes and manage conflicts with other developer's work.
+
+So if you've made the decision to try to attempt to source/version control your Umbraco implementation work, perhaps setting up a ['Git Repository'](https://en.wikipedia.org/wiki/Git) - then a frequently asked question is:
+
+### What folders and files should I **exclude** from my source control repository?
+
+There are lots of different possible variations within your working environment that will affect the best way to setup version control. It depends on whether you are:
+
+- Working with a team of developers.
+- How your development environment is setup.
+- Source control repository.
+- And also how you intend to build and deploy your solution to your target production environment (build servers, Web Deploy or good old FTP etc).
+
+However, V9 ships with a `.gitignore` file with a custom Umbraco section, which will make git ignore the files for you, if you use git, the Umbraco specific section looks like this:
```

Comment:
```suggestion
## Umbraco Cloud

If you are running your site on Umbraco Cloud - and why wouldn't you be? - then source control is very much part of the experience, have a look at the ['Technical overview of an Umbraco Cloud Environment'](../../../Umbraco-Cloud/Getting-Started/Environments/) and the information on ['Working with your Umbraco Cloud project'](../../../Umbraco-Cloud/Set-Up/#working-with-your-umbraco-cloud-project) for a steer on Source/Version Control good practices.

## Outside of Umbraco Cloud

If you are hosting your Umbraco implementation outside of Umbraco Cloud, it's generally considered good practice to set up source/version control for your site implementation files. This is especially a good idea when you are working with a team as it can help you track changes and manage conflicts with other developer's work.

So if you've made the decision to try to attempt to source/version control your Umbraco implementation work, perhaps setting up a ['Git Repository'](https://en.wikipedia.org/wiki/Git) - then a frequently asked question is:

### What folders and files should I **exclude** from my source control repository?

There are lots of different possible variations within your working environment that will affect the best way to set up version control. It depends on whether you are:

- Working with a team of developers.
- How your development environment is set up.
- Source control repository.
- And also how you intend to build and deploy your solution to your target production environment (build servers, Web Deploy or good old FTP, etc).

However, V9 ships with a `.gitignore` file with a custom Umbraco section, which will make git ignore the files for you, if you use git, the Umbraco specific section looks like this:
```